assets/js/googlesitekit/api/cache.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
+
+let storageBackend;
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @param {*} backend Backend to set for the cache.
+ */
+export const _setSelectedStorageBackend = ( backend ) => {
+	storageBackend = backend;
+};
+
+let storageKeyPrefix = 'googlesitekit_';
```

Comment:
This should become a `const`, and the name should be capitalized and exported so that other code can easily reference this prefix.